src/SupertypeLogger.ts

```diff
@@ -178,7 +186,7 @@ export class SupertypeLogger {
         }
     }
 
-    sendToLog(level, json) {
+    sendToLog(level, json, ...rawData) {
```

Comment:
where are we using `rawData` in this function?

Reply:
right now, we add some additional properties to the object that's going to get logged out for convenience (formatting, adding some timestamps, etc.). Eventually, our internal logger module will take care of that. so we want to keep our existing logs formatted, while also allowing the module logger to use an untouched version of the data that came in.

Reply:
I just mean - `sendToLog` listed as an argument of the function, but its not being used in the function?

Reply:
spoke offline yesterday, to summarize - we needed this function signature to accept three args, even though our default implementation is only utilizing two. resolution: put a code comment explaining the role of that third param.